com.unity.shadergraph/Editor/Drawing/MaterialGraphEditWindow.cs

```diff
@@ -235,7 +235,8 @@ public void ToSubGraph()
 
             var copyPasteGraph = new CopyPasteGraph(
                     graphView.selection.OfType<MaterialNodeView>().Where(x => !(x.node is PropertyNode)).Select(x => x.node as INode),
-                    graphView.selection.OfType<Edge>().Select(x => x.userData as IEdge));
+                    graphView.selection.OfType<Edge>().Select(x => x.userData as IEdge),
+                    graphView.selection.OfType<BlackboardField>().Select(x => x.userData as IShaderProperty));
```

Comment:
Doesn't work on trunk, needs a conditional import of `UnityEditor.Experimental.UIElements.GraphView` if not `UNITY_2018_1`